src/pages/patch/PatchRestartModal.tsx

```diff
@@ -28,28 +30,119 @@ import { queryString } from "utils";
 const { getArray, parseQueryString } = queryString;
 const { gray } = uiColors;
 
-interface PatchModalProps {
+interface PatchRestartContentProps {
   visible: boolean;
   onOk: () => void;
   onCancel: () => void;
   patchId?: string;
   refetchQueries: string[];
   childPatches: Partial<Patch>[];
+  setShouldAbortInProgressTasks: React.Dispatch<React.SetStateAction<boolean>>;
+  shouldAbortInProgressTasks: boolean;
+  shouldRestart: number;
+  setMutationLoading: React.Dispatch<React.SetStateAction<boolean>>;
+  setButtonDisabled: React.Dispatch<React.SetStateAction<boolean>>;
+  buttonDisabled: boolean;
 }
-export const PatchRestartModal: React.FC<PatchModalProps> = ({
+
+export const PatchRestartContent: React.FC<PatchRestartContentProps> = ({
```

Comment:
Can we move this into a separate file? This file currently exports two components. It will keep things more organized if it only exported 1 and the filename matches the name of the export